src/content/en/fundamentals/media/manipulating/files.md

```diff
@@ -0,0 +1,325 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: TBD.
+
+{# wf_updated_on: 2017-06-23 #}
+{# wf_published_on: 2017-06-23 #}
+
+# Media File Manipulation {: .page-title }
```

Comment:
I think this probably isn't "Media File Manipulation" - it's "Transforming a raw video file into an encrypted multi-bandwidth resource" or the like?